### PR TITLE
Fixes from Ev

### PR DESCRIPTION
**Pull request by whatevsss**

### CODE REVIEW

**Comment by jschneider1207** [comment]
thats not nice :c

File: KiCadProject.py [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+    # bad pracice ot use an __init__ file do more than one thing
+    def __init__(self, directory) -> Optional[None]:
+        self.directory = Path(dir)
+        if not self.directory.is_dir():
+            raise "Invalid directory path"
+        return None
+
+    def output_files(self) -> List[str]:
+        return [p for p in self.directory.iterdir() if p.is_file()]
+
```
Comment (jschneider1207) [inline KiCadProject.py]: You didnt fix the conflict right, delete this

File: KiCadProject.py [hunk 2]
```diff
@@ -15,15 +30,15 @@ def __init__(self, project_dir: Path, output_dir: Path, gerber_files: list[Gerbe
...
         output_path = dir_path / 'out'
         if not output_path.is_dir():
-            raise "Project output directory not found. Ensure the project contains a directory named 'out'."
+            return "Project output directory not found. Ensure the project contains a directory named 'out'."
 
         gerber_files = [Gerber(f) for f in output_path.glob('**/*.gbr')]
         drill_files = [Excellon(f) for f in output_path.glob('**/*.drl')]
         if gerber_files.count() == 0:
-            raise 'No gerber files found in output directory. Did you forget to plot the board?'
+            return 'No gerber files found in output directory. Did you forget to plot the board?'
```
Comment (jschneider1207) [inline KiCadProject.py]: Why return string? Its supposed to fail the script running.

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,59 @@
...
+toml==0.10.2
+tomli==2.0.1
+tqdm==4.65.0
+typing_extensions==4.5.0
+tzdata==2023.3
+urllib3==2.0.2
+Werkzeug==2.2.3
+wsproto==1.2.0
+yarl==1.9.2
+zipp==3.15.0
```
Comment (jschneider1207) [inline requirements.txt]: bruh

**Review by jschneider1207 — CHANGES_REQUESTED** [review]

File: KiCadProject.py (hunk 2 above)
Comment (whatevsss) [reply KiCadProject.py]: Syntax is incorrect

https://docs.python.org/3/tutorial/errors.html

File: requirements.txt (hunk 3 above)
Comment (whatevsss) [reply requirements.txt]: Oh whoa  have no idea why Flask is in there

**Comment by jschneider1207** [comment]
Shouldnt it be `raise Exception(“message”)`

File: requirements.txt (hunk 3 above)
Comment (whatevsss) [reply requirements.txt]: Yea tested it out twice pytest dependencies. You can use a different library.

**Pull request closed without merging**